components/script/dom/range.rs

```diff
@@ -192,11 +192,11 @@ impl Range {
             return Err(Error::HierarchyRequest);
         }
 
-        Ok((
-            first_contained_child,
-            last_contained_child,
+        Ok(ContainedChildren {
+            first_partially_contained_child: first_contained_child,
```

Comment:
Rather than doing 

```
first_partially_contained_child: first_contained_child,
```

we can also just rename the `let` bindings `first_contained_child` and `last_contained_child` in lines 164 and 174 respectively. Same in the code where this result is being used. This way the code will closely match the specification.

Reply:
you mean rename them to `first_partially_contained_child` and `last_partially_contained_child`?

Reply:
okay, I'll open a follow-up PR with the changes!

Reply:
Follow up at #33758